static/home.js

```diff
@@ -0,0 +1,69 @@
+$(document).ready(function () {
+    $.get("/repos.json?2", function (repos) {
+        const numR = repos.filter(r => r.language === "r").length;
+        const numPy = repos.filter(r => r.language === "python").length;
+        const numJs = repos.filter(r => r.language === "js" || r.language === "typescript").length;
+        const numTot = repos.length;
+
+        const pieData = {
+            labels: [
+                'R',
+                'Python',
+                "Javascript",
+                'Other'
```

Comment:
What do we have in this Other category? it's quite a lot!

Reply:
![Screenshot from 2022-08-16 15-52-33](https://user-images.githubusercontent.com/6736820/184910697-d4fc021d-213a-46d3-a203-bc390efd12cb.png)

Reply:
so mostly shell and html